pkgs/tools/misc/cope/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, perl, perlPackages }:
+
+perlPackages.buildPerlPackage rec {
+  pname = "cope";
+  version = "unstable-2024-03-27";
+
+  src = fetchFromGitHub {
+    owner = "deftdawg";
+    repo = pname;
+    rev = "ad0c1ebec5684f5ec3e8becf348414292c489175";
+    sha256 = "sha256-LMAir7tUkjHtKz+KME/Raa9QHGN1g0bzr56fNxfURQY=";
+  };
+
+  buildInputs = with perlPackages; [ EnvPath FileShareDir IOPty IOStty ListMoreUtils RegexpCommon RegexpIPv6 ];
+
+  postInstall = ''
+    mkdir -p $out/bin
+    mv $out/lib/perl5/site_perl/${perl.version}/auto/share/dist/Cope/* $out/bin/
+    rm -r $out/lib/perl5/site_perl/${perl.version}/auto
```

Comment:
shouldn't the `perl` here come from `perlPackages`?

Reply:
To be honest, I'm not sure...  I didn't author this the nix package metadata initially, so it's quite old  (almost all the things you've pointed out have evloved from what was acceptable in the days when it was originally added to NixPkgs).  

I tried to PR the fix to point at my fork of cope in Dec, but it wasn't accepted and cope was removed...  Anyway I get that if I want to re-add it I'll have to bring it up to current nix-pkg standards.  It'll be a slow process for me, since I'm still relatively new to Nix, I do welcome you're input/feedback.   I'll pick this up again when I'm back from travelling and around my main nix desktop again.